### PR TITLE
Added Repetition.py

### PR DESCRIPTION
**Pull request by amitagarwalaa57**

### CODE REVIEW

File: Repetitions.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+			else:
+				dp[i][j] = max(dp[i][j-1], dp[i-1][j]) 
+		
+	
+	return dp[n][n] 
+
+
+# Driver Program 
+if __name__=='__main__':
+	str = "ATTCGGGA"
```
Comment (codetheorem) [inline Repetitions.py]: Input should be dynamic and not pre-defined.

**Review by codetheorem — CHANGES_REQUESTED** [review]
Use `input()`.

**Comment by amitagarwalaa57** [comment]
made the change. @codetheorem

**Review by codetheorem — COMMENTED** [review]
ok thanks

**Pull request merged**